src/client.h

```diff
@@ -44,7 +44,9 @@
 # include "vstsound.h"
 #else
 # if defined ( _WIN32 ) && !defined ( JACK_REPLACES_ASIO )
-#  include "../windows/sound.h"
+//#  include "../windows/sound.h"
```

Comment:
Here you disable the old Windows sound interface?

Reply:
Yeah, I should have ifdef'd it though.